src/core/annotation.js

```diff
@@ -866,7 +866,10 @@ class Annotation {
     if (!(as instanceof Name) || !normalAppearanceState.has(as.name)) {
       return;
     }
-    this.appearance = normalAppearanceState.get(as.name);
+    const appearance = normalAppearanceState.get(as.name);
```

Comment:
Maybe it'll be simpler to have a getter like `getAsStream` which will return a `Stream` or `null`. Wdyt ?

Reply:
I don't know if makes sense to extend the `Dict`-class with a new method like that, since this pattern doesn't seem all that common and we often have different code-paths depending on the data-type returned or simply want to throw an Error if it's not a Stream.